Project.toml

```
@@ -11,10 +11,10 @@ SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 ToeplitzMatrices = "c751599d-da0a-543b-9d20-d0a503d91d24"
 
 [compat]
-FFTW = "1.2"
-OffsetArrays = "1.0"
-SpecialFunctions = "1"
-ToeplitzMatrices = "0.6"
+FFTW = "1"
+OffsetArrays = "1"
+SpecialFunctions = "1,2"
+ToeplitzMatrices = "0.6,0.7,0.8"
 julia = "1.4"
```

```suggestion
julia = "1.10"
```

If we bump the CI Julia version, we should also do so here.